docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -54,10 +56,359 @@ Integrations Server. To do this:
 
 NOTE: When stopping incoming requests, host-agent replicas back off and retry connecting to the {ecloud} endpoint at increasing time intervals.
 
+[discrete]
+[[profiling-migrate-data]]
+=== Migrate existing profiling data
+
+When upgrading to version 8.9, you can use a migration script to migrate existing data if you meet the following requirements:
+
+* The cluster has been upgraded from 8.8 to 8.9.
+* The migration is performed on a Linux workstation with `jq` and `curl` installed.
+
+If you meet these requirements, follow the steps below to run the migration script. If not, we suggest <<profiling-delete-data, deleting your existing profiling data>> and <<profiling-from-scratch, setting up Universal Profiling from scratch>>.

```

Comment:
At this point I went looking through the prior text to find a link to the script for about 30 seconds before realizing that it follows below -- might just be me though.

Reply:
Good point. How about we reword this sentence to:

> If you meet these requirements, follow the steps below to migrate your data.

This just mentions the goal ("migrate the data") and directs the user to the following steps. The solution (i.e. the script) is only mentioned in the steps and at that point it's hopefully easier to spot. One note: I would have wished we could just provide a download link instead of rendering the script inline but the inline approach seems to be the established practice.

Reply:
Sounds good!